### PR TITLE
[tree widget] hide orphan sv and svg

### PR DESCRIPTION
**Pull request by chejennifer**
- When creating search index and parent map, ignore orphan sv and svg (ones not attached to the root) so that they don't show up in search results

### CODE REVIEW

File: internal/server/statvar/statvar_hierarchy_util.go [hunk 1]
```diff
@@ -61,21 +61,38 @@ func GetRawSvg(ctx context.Context, baseTable *cbt.Table) (
 	return svgResp.StatVarGroups, nil
 }
 
-// BuildParentSvgMap gets the mapping of svg/sv id to the parent svg for that svg/sv.
+// BuildParentSvgMap gets the mapping of svg/sv id to the parent svg for that
+// svg/sv. Only gets the parent svg that have a path to the root node.
 func BuildParentSvgMap(rawSvg map[string]*pb.StatVarGroupNode) map[string][]string {
 	parentSvgMap := map[string][]string{}
```
Comment (pradh) [inline internal/server/statvar/statvar_hierarchy_util.go:67]: nit: ancestorSvgMap ?

**Review by pradh — APPROVED** [review]
Nice!

File: internal/server/statvar/statvar_hierarchy_util.go (hunk 1 above)
Comment (chejennifer) [reply internal/server/statvar/statvar_hierarchy_util.go:67]: This map only holds the direct parent svgs of each sv or svg, so I'm not sure that ancestorSvgMap will make more sense?

Comment (pradh) [reply internal/server/statvar/statvar_hierarchy_util.go:67]: Ah, I see... sorry i misread it.

**Pull request merged**